### PR TITLE
db.query.js: queries for creating tables, creating entries, etc

### PR DESCRIPTION
**Pull request by joie-zhang**
also contains dbSetup.js, which was used to populate the problems database using the leetcode_problems.csv file

### CODE REVIEW

**Review by brok3turtl3 — APPROVED** [review]
Great work! Really nicely comment files.

**Review by sifulsidd — APPROVED** [review]
Great setup!

**Review by Ericesposito — APPROVED** [review]
Great job Joie, the DB looks great!

File: server/models/dbSetup.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+dotenv.config();
+
+import pkg from 'pg';
+const { Client } = pkg;
+import fs from "fs";
+import abc from 'pg-copy-streams';
+const { from } = abc;
+
+// Replace with your actual connection URI
+const connectionUri = 'postgres://guuyjnbs:qpnYB-UNCbYV1MZ4rvOadoq48CY2tpSO@bubble.db.elephantsql.com/guuyjnbs';
```
Comment (joie-zhang) [inline server/models/dbSetup.js]: This should be replaced with process.env.PG_URI

**Review by paglenn — APPROVED** [review]
nice!

**Review by paglenn — APPROVED** [review]

**Pull request merged**